### DISCUSSION TITLE
Progress Report 31 May - 3 June

### DISCUSSION THREAD

**Discussion by ian-noaa** [NOAA-GSL/unified-graphics#28] [Progress Report]
🚴🏻 **Progress**
- Our container images are now publicly available in a [GitHub Container Registry associated with this project](https://github.com/orgs/NOAA-GSL/packages?repo_name=unified-graphics)

😖 **Regressions**
- #27 
- #23 

💬 **To Discuss**
- Contents of the NetCDF diag files
- Where we want to commit our Kubernetes manifests

📆 **Next Week**
- Push images into VLab's container image registry
- Work with VLab on security scanning
- Work with VLab on deploying images into the Kubernetes cluster
- Continue exploring NetCDF diag files

**Comment by esheehan-gsl** [comment]
@ian-noaa nice work on the CI

**Reply by ian-noaa** [reply]
Thanks!
